blockchain/genesis/genesis.go

```diff
@@ -53,6 +53,7 @@ func defaultConfig() Genesis {
 			BeringBlockHeight:       1512001,
 			CookBlockHeight:         1641601,
 			DardanellesBlockHeight:  1816201,
+			DaytonaBlockHeight:      3168001,
```

Comment:
I think this is not at start of an epoch. You may use 360 block pre epoch to calculate, now we have 720 block pre epoch

Reply:
i used 720 to calculate at first, but found it is not multiple of 720, then change it. 

It's b/c Dardarnelle is not a multiple of 720, which i failed to realize

added comments in lastest push